### DISCUSSION TITLE
How to interact with an already running session of AEDT through pyaedt

### DISCUSSION THREAD

**Discussion by gnimdu** [ansys/pyaedt#3221] [Q&A] [answered]
launching aedt through pyaedt in graphical mode generally take some time. Is it possible to start a local session of aedt and then test various script in this already running session without having to relaunch a session through pyaedt?

**Comment by fwlfHS** [comment, marked as answer]
Use `new_desktop_session=False` when initializing PyAEDT, this will connect to a running session.

**Reply by gnimdu** [reply]
thank you. works perfectely!
